actionsystemtest/actionsystemtest.gemspec

```diff
@@ -0,0 +1,25 @@
+version = File.read(File.expand_path("../../RAILS_VERSION", __FILE__)).strip
+
+Gem::Specification.new do |s|
+  s.platform    = Gem::Platform::RUBY
+  s.name        = "actionsystemtest"
+  s.version     = version
+  s.summary     = "Acceptance test framework for Rails."
+  s.description = "Test framework for testing web applications by simulating how users interact with your application."
+
+  s.required_ruby_version = ">= 2.2.2"
+
+  s.license = "MIT"
+
+  s.author   = ["Eileen Uchitell", "David Heinemeier Hansson"]
```

Comment:
I noticed that, still working on a bunch of stuff. I'll let you know when I'm ready for more review.